---
post_url: /t/project-1-llm-based-automation-agent-discussion-thread-tds-jan-2025/164277/486
---
[@carlton](/u/carlton) [@Jivraj](/u/jivraj)

Hi ,

I am trying to build using both docker and podman but it failed on both. I have watched many videos trying to resolve this adn also chatgpt in order to resolve the issue but it seems to persist. I even uninstalled and reinstalled both podman and doceker multiple times but no help.

When i run command docker build -t \_\_\_\_\_\_\_\_\_\_\_ .

the error that comes is :

Dockerfile:2
------------

1 | # Use a lightweight Python image 
2 | >>> FROM python:3.12-slim 
3 | 
4 | # Set the working directory in the container
--------------------------------------------------------------------------------------------------------------------------

ERROR: failed to solve: python:3.12-slim: failed to resolve source metadata for [Docker Hub Container Image Library | App Containerization](http://docker.io/library/python:3.12-slim:) failed to copy: httpReadSeeker: failed open: failed to do request: Get “<https://docker-images-prod.6aa30f8b08e16409b46e0173d6de2f56.r2.cloudflarestorage.com/registry-v2/docker/registry/v2/blobs/sha256/6f/6f3c6367c5a38963f84310cbb24dfcfbddab1dad40cff18afb8fe89098891f08/data?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=f1baa2dd9b876aeb89efebbfc9e5d5f4%2F20250215%2Fauto%2Fs3%2Faws4_request&X-Amz-Date=20250215T192245Z&X-Amz-Expires=1200&X-Amz-SignedHeaders=host&X-Amz-Signature=ed37cf0c346e2ed440f29638ec43ce66640bdc7d285e7be7bf25c308c46fd6b1>”: dialing [docker-images-prod.6aa30f8b08e16409b46e0173d6de2f56.r2.cloudflarestorage.com:443](http://docker-images-prod.6aa30f8b08e16409b46e0173d6de2f56.r2.cloudflarestorage.com:443) container via direct connection because static system has no HTTPS proxy: connecting to [docker-images-prod.6aa30f8b08e16409b46e0173d6de2f56.r2.cloudflarestorage.com:443](http://docker-images-prod.6aa30f8b08e16409b46e0173d6de2f56.r2.cloudflarestorage.com:443): dial tcp: lookup [docker-images-prod.6aa30f8b08e16409b46e0173d6de2f56.r2.cloudflarestorage.com](http://docker-images-prod.6aa30f8b08e16409b46e0173d6de2f56.r2.cloudflarestorage.com): no such host

Even tried getting python:3.12-slim separatly and trying again but that also didn’t work.  
I think there is some problem in getting python:3.12-slim as the build always stops at this.

on asking ChatGPT it shows that some DNS or network issue is there. I even tried all the remedy that was provided on creating custom network etc. but this was also of no use

Kindly help me finding solution to this and pls mention any other assistance I may require to get this running

Thank You.  
Regards,  
Aagman